templates/documentation/vod/video-object.md

```diff
@@ -18,6 +18,18 @@ The video object has properties and metadata that you can create and update, bel
 
 The video object is a powerful feature that you can utilize in order to achieve many use cases by leveraging the parameter that is offered with the object.
 
+### Video ID
+
+The video ID is a unique identifier for the video object that you create. Use it to identify a specific video object through the API. For example, you need the video ID when you [upload a video](https://docs.api.video/reference/api/Videos#upload-a-video), [retrieve a video object](https://docs.api.video/reference/api/Videos#retrieve-a-video-object), or [update a video object](https://docs.api.video/reference/api/Videos#update-a-video-object). For a full list of API operations, check out the [`/videos` endpoint](https://docs.api.video/reference/api/Videos) in the API reference.
+
+You can retrieve the video ID for a video object via these methods:
+
+* when creating a video object, the API sends the `videoId` in the response payload
+* when listing all video objects associated with your current workspace, the API sends the `videoId` for each video object in the response payload
```

Comment:
instead of `sends` I would use `returns`

Reply:
Done, for both `sends` :)